### ISSUE TITLE
NTR: Purulent drainage from Stensen duct (Face & neck)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7827]
New term request Purulent drainage from Stensen duct (seidel_200525135116): Pus; suppurative; purulence; parotitis; parotid duct; salivary  UMLS: 0030583

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000178
- label: Purulent drainage from Stensen duct
- parents: [Abnormal parotid gland morphology (HP:0000197)](https://hpo.jax.org/app/browse/term/HP:0000197)
- pmid: 
- def: Flow of pus (purulence) from the Stensen duct. The Stensen duct is also known as the parotid duct and is the route that saliva takes from the major salivary gland (parotid gland) to the mouth.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7827
- synonyms:
